space-cli/plugins.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"plugin"
+
+	"github.com/spf13/cobra"
+)
+
+func getplugin() (*cobra.Command, error) {
+	mod := fmt.Sprintf("%s/cmd_%s.so", getSpaceCliDirectory(), latestVersion)
+	plug, err := plugin.Open(mod)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	_, err = plug.Lookup("")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	// //var greeter Greeter
+	// greeter, ok := symGreeter.(Greeter)
```

Comment:
As I said. The type will be the functions. Name the variable `getRootCommand` not `greeter`